Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CATracker.h

```diff
@@ -10,16 +10,18 @@
 #define ALIITSUCATRACKER_H
 
 #include <vector>
+#include <array>
 
 #include "ITSReconstruction/CAaux.h"
 #include "ITSReconstruction/CATrackingStation.h"
 #include "DetectorsBase/Track.h"
 
-using std::vector;
-
 namespace AliceO2 {
   namespace ITS {
     namespace CA {
+      using std::vector;
```

Comment:
Bottom line, IMHO, is: if you want a type alias in an header file, use a class level typedef.